vowpalwabbit/cb_adf.cc

```diff
@@ -485,6 +492,7 @@ base_learner* cb_adf_setup(VW::setup_base_i& stack_builder)
   bool rank_all;
   float clip_p;
   bool no_predict;
+  bool ccb = false;
```

Comment:
This reduction knowing about ccb is a big break of encapsulation.

Reply:
Would it make sense to use the same condition as `conditional_contextual_bandit` setup, i.e. `if (options.was_supplied("quiet") && !options.was_supplied("p")) { data->no_pred = true; }` and have a no-pred boolean here too?

Reply:
It can all be handled in ccb so I removed this as well